Core/Libraries/Source/WWVegas/WWLib/wwstring.cpp

```diff
@@ -197,8 +196,8 @@ StringClass::Free_String (void)
 {
 	if (m_Buffer != m_EmptyString) {
 
-		unsigned buffer_base=reinterpret_cast<unsigned>(m_Buffer-sizeof (StringClass::_HEADER));
-		unsigned temp_base=reinterpret_cast<unsigned>(m_TempStrings+MAX_TEMP_BYTES*MAX_TEMP_STRING);
+		unsigned long buffer_base=reinterpret_cast<unsigned long>(m_Buffer-sizeof (StringClass::_HEADER));
+		unsigned long temp_base=reinterpret_cast<unsigned long>(m_TempStrings+MAX_TEMP_BYTES*MAX_TEMP_STRING);
```

Comment:
Is this meant to take the size of pointer? long is 4 bytes on Windows x64. See:

https://en.cppreference.com/w/cpp/language/types
https://en.cppreference.com/w/cpp/language/types#Data_models

Reply:
Perhaps replicate the c++11 stdint types in a "Dependencies/Utility/stdint_adapter.h"

Then include that adapter in BaseTypeCore.h and always.h

https://en.cppreference.com/w/cpp/header/cstdint

Reply:
Yes can be separate.